### PR TITLE
README.md: add Comet ML logo

### PR DESCRIPTION
**Pull request by kobindra**
In support of 0.10.0 release!

### CODE REVIEW

**Review by mvpatel2000 — APPROVED** [review]
LGTM! Approving to unblock assuming this was cleared to be added. If it wasn't, defer to Hanlin/Bandish.

**Comment by lupesko** [comment]
How is this badge useful for Composer users?
The comet_ml badge is rendered in a row with Composer badges, the comet_ml badge looks like it does not fit there.
As a note, there is no badge for other supported tools such as wandb.

I suggest to rethink this. It would be better to have this in a section with "supported tools".

**Comment by kobindra** [comment]
No problem. I'm closing this PR.

**Pull request closed without merging**
